src/wrapper/objects/auto_byte_array.rs

```diff
@@ -0,0 +1,94 @@
+use jni_sys::{jbyte, JNI_ABORT};
+use log::debug;
+
+use crate::{objects::JObject, JNIEnv};
+
+/// ReleaseMode
+///
+/// This defines the release mode of AutoByteArray (and AutoPrimitiveArray) resources, and
+/// related release array functions.
+#[derive(Clone, Copy)]
+pub enum ReleaseMode {
+    /// Copy back the content and free the elems buffer.
+    Copy = 0,
+    /// Free the buffer without copying back the possible changes.
+    NoCopy = JNI_ABORT as isize,
```

Comment:
Yes, I can change the `enum` representation to avoid the cast:
```
#[derive(Clone, Copy)]
#[repr(i32)]
pub enum ReleaseMode {
    /// Copy back the content and free the elems buffer.
    Copy = 0,
    /// Free the buffer without copying back the possible changes.
    NoCopy = JNI_ABORT,
}
```
Will do.